dispatchrouter/qpid/qdrouterd-with-broker.conf

```diff
@@ -0,0 +1,147 @@
+router {
```

Comment:
Yes. I was having a problem with 0.7.0 with JSON config, so I used this. We can remove it know as this is also fixed with 0.8.0.